lib/game/game.dart

```diff
@@ -130,6 +131,21 @@ abstract class Game {
     );
   }
 
+  /// Utility method to load and cache the image for a [SpriteBatch] based on its options
+  Future<SpriteBatch> loadSpriteBatch(
```

Comment:
Since SpriteBatch is not so common like Sprite and SpriteAnimations are, I think this could be into an extension inside the spritebatch.dart, just like it was done on Parallax.

Also, we should have the game cache being used here, with a fallback to the global cache when omitted.

Reply:
Let me update that :-)

Reply:
> Also, we should have the game cache being used here, with a fallback to the global cache when omitted.

Not sure what you meant with this, but I updated the PR to use the extension, so if you could have another look.